src/pages/country/styles.scss

```diff
@@ -17,13 +17,14 @@
         letter-spacing: -2.16px;
         line-height: 42px;
         text-align: center;
+        margin-bottom: 35px;
       }
 
       p {
-        padding: 0 180px;
         font-weight: 300;
```

Comment:
This font-weight should be on the body or at least in the base.scss for paragraphs.